src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -1037,6 +1040,9 @@ public void CSharp_VerifyIDECodeStyleOptionsAreConfigurable()
 # IDE0200, PreferMethodGroupConversion
 csharp_style_prefer_method_group_conversion = true
 
+# IDE0210
+No editorconfig based code style option
```

Comment:
I'm not familiar with these tests, but isn't `dotnet_style_prefer_foreach_explicit_cast_in_source` an editorconfig based code style option? Is this only for `csharp_style_XXX`?

Reply:
@davidwengier Good catch! The option passed to `AbstractBuiltInCodeStyleDiagnosticAnalyzer` ctor is `null`. I'll comment on the exact line now.